app/templates/people/person/personal-information/edit.hbs

```diff
@@ -0,0 +1,101 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuToolbar>
+        <AuToolbarGroup>
+          <AuHeading @level="1" @skin="2">
+            Bewerk Persoonlijke gegevens
+          </AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuHeading @level="2" @skin="3">
+        {{@model.givenName}} {{@model.familyName}}
+      </AuHeading>
+    </div>
+
+    <form id="edit-personal-information-form" aria-label="edit-personal-information-form" {{on "submit" (perform
+      this.save)}}>
+
+      <div class="au-o-flow au-o-flow--large">
+
+        <div>
+          <AuToolbar class="au-u-margin-bottom-small">
+            <AuToolbarGroup>
+              <AuHeading @level="3" @skin="5">Persoonlijke gegevens</AuHeading>
+            </AuToolbarGroup>
+          </AuToolbar>
+          <AuCard as |c|>
+            <c.content>
+              <div class="au-o-grid">
+                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                  <div class="au-o-grid">
+                    <div class="au-o-grid__item au-u-1-4">
+                      <AuLabel for="person-given-name">
+                        Voornaam
+                      </AuLabel>
+                    </div>
+                    <div class="au-o-grid__item au-u-3-4">
+                      <AuInput @value={{@model.givenName}} @width="block" id="person-given-name" />
```

Comment:
using `@model` as a record is confusing. I would either return an object with a person key from the model hook (so this becomes `@model.person.givenName` or alias the model in the controller so it becomes `this.person.givenName`. That will make things way more readable imo.